db/upgrade-jambonz-db.js

```diff
@@ -173,7 +173,8 @@ const sql = {
     'ALTER TABLE applications ADD COLUMN fallback_speech_recognizer_vendor VARCHAR(64)',
     'ALTER TABLE applications ADD COLUMN fallback_speech_recognizer_language VARCHAR(64)',
     'ALTER TABLE applications ADD COLUMN fallback_speech_recognizer_label VARCHAR(64)',
-    'ALTER TABLE sip_gateways ADD COLUMN pad_crypto BOOLEAN NOT NULL DEFAULT 0,',
+    'ALTER TABLE sip_gateways ADD COLUMN pad_crypto BOOLEAN NOT NULL DEFAULT 0',
+    'ALTER TABLE sip_gateways MODIFY port INTEGER DEFAULT 5060'
```

Comment:
I don't think we should have a default at all

Reply:
Fixed, remove default 5060 for sip_gateway.port

Reply:
I fixed both upgrade-jambonz-db.js and also in jambonz.sqs